server/src/main/java/org/apache/iotdb/db/query/reader/resourceRelated/CachedUnseqResourceMergeReader.java

```diff
@@ -35,8 +35,8 @@ public CachedUnseqResourceMergeReader(List<Chunk> chunks, TSDataType dataType)
     super(dataType);
     int priorityValue = 1;
     for (Chunk chunk : chunks) {
-      ChunkReader chunkReader = new ChunkReaderWithoutFilter(chunk);
-      addReaderWithPriority(new CachedDiskChunkReader(chunkReader), priorityValue++);
+      AbstractChunkReader AbstractChunkReader = new ChunkReader(chunk, null);
```

Comment:
The first letter of variable should be lowercase

Reply:
fixed